composer.json

```diff
@@ -15,14 +16,16 @@
         "goaop/framework": "^2.1",
         "kickasscommerce/moltin": "dev-master",
         "cache/filesystem-adapter": "^0.3.3",
-        "symfony/console": "^3.2"
+        "symfony/console": "^3.2",
+        "kickasscommerce/authentication": "*@dev"
     },
     "require-dev": {
         "phpunit/phpunit": "*"
     },
     "autoload": {
         "psr-4": {
             "KickAss\\Commerce\\": "src/",
+            "KickAss\\Authentication\\": "src/Authentication",
```

Comment:
I could not get it to pick the auto loader from the sup repo but maybe you know the magic!